compatibility/jersey/src/main/java/org/projectnessie/tools/compatibility/jersey/JerseyServer.java

```diff
@@ -172,20 +174,24 @@ private static void withClass(
 
   private static void withJavaxClass(
       String javaxClassName, Consumer<Class<?>> whenClassExists, boolean mandatory) {
-    try {
-      Class<?> clazz = Thread.currentThread().getContextClassLoader().loadClass(javaxClassName);
-      whenClassExists.accept(clazz);
-    } catch (NoClassDefFoundError | ClassNotFoundException e) {
+    Iterator<String> classNames =
+        List.of(
+                javaxClassName,
+                javaxClassName.replace("restjavax.", "restjakarta."),
+                javaxClassName.replace("restjavax.", "rest."))
```

Comment:
How does it work with the old `rest` stuff when extensions are linked to the `jakarta` SPI.

Reply:
This is only class-loading in a different class-loader using the code from the old versions.
Seems to work, since the compatibility-tests still pass.

Reply:
Does Jersey support both sets of annotations?

Reply:
Not sure I understand the background of the question? It works.

Reply:
fair enough